# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/rocprofiler/hotfix-git-submodule.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- rocprofiler/plugin/perfetto/CMakeLists.txt.vanilla	2025-03-21 19:31:34.152624991 +0200
+++ rocprofiler/plugin/perfetto/CMakeLists.txt	2025-03-21 19:31:55.452720548 +0200
@@ -1,10 +1,3 @@
-rocprofiler_checkout_git_submodule(
-            RECURSIVE
-            TEST_FILE "sdk"
-            RELATIVE_PATH perfetto
-            WORKING_DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}
-            REPO_URL https://github.com/google/perfetto.git
-            REPO_BRANCH "releases/v44.x")
 
 include_directories(perfetto/sdk)
 
